### PR TITLE
Update requesthandlersettings.md

### PR DESCRIPTION
**Pull request by aguyfromdenmark**
Adds note telling the reader that they will need to re-save and publish nodes to see effects of changes in the char collection.

### CODE REVIEW

**Review by PratheesUmbraco — COMMENTED** [review]
Also refresh the database cache before you republish for it to work.

**Comment by sofietoft** [comment]
Thanks for the PR @aguyfromdenmark ! 💪 
I've made some small adjustments to resolve the warnings.

Thanks for the input @PratheesUmbraco - how would one go about refreshing the database cache?
Perhaps you know of another article or similar, that we could refer to?

**Comment by PratheesUmbraco** [comment]
@sofietoft im afraid we do not have this documentation :/ Im referring to this part of the Settings section
image

**Comment by sofietoft** [comment]
Ah, I see!

I'll add a task to have that section documented.
For now, let's add a line about it to the note here instead.

File: umbraco-cms/reference/configuration/requesthandlersettings.md [hunk 1]
```diff
@@ -136,7 +136,11 @@ This setting tells Umbraco to use the default character replacements. If you don
 
 ### Char collection
 
-This settings contains a lot of obejcts with a **`"Char"`** and **`"Replacement`** key, each of these object holds a character that should be replaced, and what it should be replaced with.
+This settings contains objects with a **"Char"** and **"Replacement** key. Each of these objects holds a character that should be replaced and what it should be replaced with.
+
+{% hint style="info" %}
+When adding or removing entries from the collection, you need to re-save and publish any node where you want the change reflected. You can also use **Publish with descendants...** on the root node to update the URL for all descendants.
```
Comment (sofietoft) [inline umbraco-cms/reference/configuration/requesthandlersettings.md:142]:
```suggestion
When managing entries from the collection, you need to do a couple of things for the change to be reflected: 

1. Refresh the database cache from the "Published Status" tab in the **Settings** section of the backoffice.
2. Re-save and publish any node where you want the change reflected. 
  a. You can also use **Publish with descendants...** on the root node to update the URL for all descendants.
```

**Review by sofietoft — COMMENTED** [review]
@aguyfromdenmark , @PratheesUmbraco - please see my suggestion to update the note to include both steps you've mentioned.

**Comment by sofietoft** [comment]
I've remade this PR for the new docs structure: https://github.com/umbraco/UmbracoDocs/pull/4963
I'll close this PR.

**Pull request closed without merging**